samples/coroutines/coroutines.gdnlib

```diff
@@ -7,12 +7,10 @@ reloadable=true
 
 [entry]
 
-OSX.64="res://libkotlin.dylib"
-Windows.64="res://kotlin.dll"
-X11.64="res://libkotlin.so"
+Windows.64="res://kotlin/build/bin/windows/releaseShared/kotlin.dll"
```

Comment:
On the games sample project we link the debug version and here the release version. I think we shouldlink the debug version both times as they are samples not "finished" games and its more consistent.

Reply:
@Wavesonics, can you rebase on master and add requested changes so that we get that merged :)

Reply:
Oh did not see it's coroutine module, it is compiled as release in gradle build script. Can you set it in debug for the moment ? We will add a target parameter to define release or debug.